src/blocks/row/styles/editor.scss

```diff
@@ -352,6 +352,7 @@
 
 		> span > .components-resizable-box__handle {
 			display: block;
+			right: -0.7em !important;
```

Comment:
@richtabor. I couldn't see a way around this, in order to get the resizable handle to appear in the desired position. Any idea how I can avoid this?

Reply:
May not be. Positioning those handles is so so hacky :/